src/mpid/ch4/netmod/ofi/ofi_capability_sets.h

```diff
@@ -422,6 +425,61 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_OFI_get_set_number(const char *set_name)
+#define MPIDI_OFI_ENABLE_AV_TABLE_RXM              MPIDI_OFI_OFF
+#define MPIDI_OFI_ENABLE_SCALABLE_ENDPOINTS_RXM    MPIDI_OFI_OFF
+#define MPIDI_OFI_ENABLE_STX_RMA_RXM               MPIDI_OFI_OFF
+#define MPIDI_OFI_ENABLE_MR_SCALABLE_RXM           MPIDI_OFI_OFF
+#define MPIDI_OFI_ENABLE_TAGGED_RXM                MPIDI_OFI_ON
+#define MPIDI_OFI_ENABLE_AM_RXM                    MPIDI_OFI_ON
+#define MPIDI_OFI_ENABLE_RMA_RXM                   MPIDI_OFI_ON
+#define MPIDI_OFI_ENABLE_ATOMICS_RXM               MPIDI_OFI_OFF
+#define MPIDI_OFI_MAX_ENDPOINTS_RXM                MPIDI_OFI_MAX_ENDPOINTS_REGULAR
+#define MPIDI_OFI_MAX_ENDPOINTS_BITS_RXM           MPIDI_OFI_MAX_ENDPOINTS_BITS_REGULAR
+#define MPIDI_OFI_FETCH_ATOMIC_IOVECS_RXM           1
+#define MPIDI_OFI_ENABLE_DATA_AUTO_PROGRESS_RXM    MPIDI_OFI_OFF
+#define MPIDI_OFI_ENABLE_CONTROL_AUTO_PROGRESS_RXM MPIDI_OFI_OFF
+#define MPIDI_OFI_ENABLE_PT2PT_NOPACK_RXM          MPIDI_OFI_ON
+#define MPIDI_OFI_CONTEXT_BITS_RXM                 (16)
+#define MPIDI_OFI_SOURCE_BITS_RXM                  (24)
+#define MPIDI_OFI_TAG_BITS_RXM                     (20)
+#define MPIDI_OFI_MAJOR_VERSION_RXM                 1
+#define MPIDI_OFI_MINOR_VERSION_RXM                 5
+
+#ifdef MPIDI_CH4_OFI_USE_SET_RXM
+#define MPIDI_OFI_ENABLE_RUNTIME_CHECKS         MPIDI_OFI_OFF
+#define MPIDI_OFI_ENABLE_DATA                   MPIDI_OFI_ENABLE_DATA_RXM
+#define MPIDI_OFI_ENABLE_AV_TABLE               MPIDI_OFI_ENABLE_AV_TABLE_RXM
+#define MPIDI_OFI_ENABLE_SCALABLE_ENDPOINTS     MPIDI_OFI_ENABLE_SCALABLE_ENDPOINTS_RXM
+#define MPIDI_OFI_MAX_ENDPOINTS                 MPIDI_OFI_MAX_ENDPOINTS_RXM
+#define MPIDI_OFI_MAX_ENDPOINTS_BITS            MPIDI_OFI_MAX_ENDPOINTS_BITS_RXM
+#define MPIDI_OFI_ENABLE_STX_RMA                MPIDI_OFI_ENABLE_STX_RMA_RXM
+#define MPIDI_OFI_ENABLE_MR_SCALABLE            MPIDI_OFI_ENABLE_MR_SCALABLE_RXM
+#define MPIDI_OFI_ENABLE_TAGGED                 MPIDI_OFI_ENABLE_TAGGED_RXM
+#define MPIDI_OFI_ENABLE_AM                     MPIDI_OFI_ENABLE_AM_RXM
+#define MPIDI_OFI_ENABLE_RMA                    MPIDI_OFI_ENABLE_RMA_RXM
+#define MPIDI_OFI_ENABLE_ATOMICS                MPIDI_OFI_ENABLE_ATOMICS_RXM
+#define MPIDI_OFI_FETCH_ATOMIC_IOVECS           MPIDI_OFI_FETCH_ATOMIC_IOVECS_RXM
+#define MPIDI_OFI_IOVEC_ALIGN                   (1)
+#define MPIDI_OFI_ENABLE_DATA_AUTO_PROGRESS     MPIDI_OFI_ENABLE_DATA_AUTO_PROGRESS_RXM
+#define MPIDI_OFI_ENABLE_CONTROL_AUTO_PROGRESS  MPIDI_OFI_ENABLE_CONTROL_AUTO_PROGRESS_RXM
+#define MPIDI_OFI_ENABLE_PT2PT_NOPACK           MPIDI_OFI_ENABLE_PT2PT_NOPACK_RXM
+#define MPIDI_OFI_PROTOCOL_MASK                 (0xE000000000000000ULL)
+#define MPIDI_OFI_CONTEXT_MASK                  (0x0FFFF00000000000ULL)
+#define MPIDI_OFI_SOURCE_MASK                   (0x00000FFFFFF00000ULL) /* RxM does not support immediate data
+                                                                           so this field needs to be available */
```

Comment:
One nitpick from the code cleanup script. This line should have an `*` at the beginning under the previous line's `*`.

Reply:
I ran `maint/code_cleanup.sh`, but forgot to update the PR with changes.

Thanks. I've updated the PR